features/FEATURE_LWIP/lwip-interface/lwip/src/netif/ppp/lwip_auth.c

```diff
@@ -751,6 +751,10 @@ void link_established(ppp_pcb *pcb) {
     if (!doing_multilink) {
 	for (i = 0; (protp = protocols[i]) != NULL; ++i)
 	    if (protp->protocol != PPP_LCP
+#if PPP_IPV4_SUPPORT && PPP_IPV6_SUPPORT
```

Comment:
This feels like it's in the wrong layer. Shouldn't a suppression check be in ip[v6]cp_lowerup?